package.json

```diff
@@ -1,20 +1,24 @@
 {
     "name": "@itslearning/protomorph",
     "description": "Shared build config for frontend applications",
-    "version": "8.3.0",
+    "version": "8.4.0",
     "author": "Gavin King <gavin.king@itslearning.com>",
     "license": "MIT",
     "scripts": {
         "build": "webpack-cli",
         "dev": "webpack-cli --watch",
         "test": "yarn build && karma start"
     },
+    "browserslist": [
+        "last 4 versions"
```

Comment:
If I'm not mistaken this will give us IE8 and newer support.

Reply:
I am afraid that this is going to add a loot extra css code that we dot not need

Reply:
Perhaps http://browserl.ist/?q=last+2+versions%2C+not+dead ?

Reply:
ie 11 is supported by default I guess